packages/tables/src/examples/overflow-menu.md

```diff
@@ -0,0 +1,98 @@
+```jsx
+const {
+  zdFontSizeBeta,
+  zdFontWeightSemibold,
+  zdSpacingSm
+} = require('@zendeskgarden/css-variables');
+const { Menu, Item } = require('@zendeskgarden/react-menus');
+
+const StyledCaption = styled(Caption)`
+  font-size: ${zdFontSizeBeta};
+  font-weight: ${zdFontWeightSemibold};
+  margin-bottom: ${zdSpacingSm};
+`;
+
+const OverflowMenu = () => (
+  <Menu
+    onChange={selectedKey => alert(selectedKey)}
+    placement="bottom-end"
```

Comment:
Menu positioning should be corrected (for all table densities) as seen in https://garden.zendesk.com/react-components-archive/#!/Table

Reply:
> Menu positioning should be corrected

There was some positioning mixups in my example. The examples now correctly:

- Apply the popperModifier to use the `viewport` as it's boundary element (not the `<tbody>`)
- Applies the necessary margin changes